client/shared/src/search/stream.ts

```diff
@@ -579,21 +579,18 @@ export function getRepositoryUrl(repository: string, branches?: string[]): strin
 }
 
 export function getRevision(branches?: string[], version?: string): string {
-    let revision = ''
-    if (branches) {
-        const branch = branches[0]
-        if (branch !== '') {
-            revision = branch
-        }
-    } else if (version) {
-        revision = version
+    if (branches && branches.length > 0) {
```

Comment:
I ain't know TS programmer but I think this can be shorted with the elvis operator:
```suggestion
    if (branches?.length > 0) {
```

Reply:
not a TS programmer either, but I believe typescript doesn't let use compare "undefined" to a number literal, which would happen if branches is undefined.